glitter_news/views.py

```diff
@@ -44,6 +44,12 @@ class PostDetailView(GlitterDetailMixin, DateDetailView):
     month_format = '%m'
     date_field = 'date'
 
+    def get_allow_future(self):
+        """ Only superusers can edit future posts. """
+        if self.request.user.is_superuser:
```

Comment:
Think we need to open this to users other than superusers.

Can we change this to something like:

```python
self.request.user.has_perm('glitter_news.edit_post')
```

Or:

```python
qs = self.get_queryset()
self.request.user.has_perm(
    '{}.edit_{}'.format(qs.model._meta.app_label, qs.model._meta.model_name
)
```

Up to you (and test the second one, I think that's the right format for it).